### ISSUE TITLE
Its a virus or no

### ISSUE THREAD

**Issue by Bruhman20** [danielkrupinski/VAC-Bypass-Loader#59]
Keeps deleting when opening the file the windows defender keeps deleting it at it said severe type of file trojan

**Comment by Bruhman20** [comment]
![aaeaeaeaea](https://user-images.githubusercontent.com/71585703/93706780-0c469b80-fb54-11ea-82f2-72369fa75bbe.JPG)
why?

**Comment by Bruhman20** [comment]
is this a virus?

**Comment by bytem1** [comment]
It is fucking safe bro.

**Comment by techzin** [comment]
This will modify files from Steam and Windows Regedit, so...

**Comment by rafalohaki** [comment]
make ur pc dissapear

**Comment by xcfrg** [comment]
Yes, it's a virus it takes your steam credentials (thats why it hook into the steam process). I lost my steam account that was worth 1k in csgo skins. Do not trust!11!111!111

**Comment by gmgm115** [comment]
i think it is a virus, weird
